### PR TITLE
FIX: Bug in _list_outputs function

### PR DESCRIPTION
**Pull request by mattfeld**
Small bugfix in the FSLXCommand _list_outputs function

### CODE REVIEW

File: nipype/interfaces/fsl/dti.py [hunk 1]
```diff
@@ -222,7 +222,7 @@ def _list_outputs(self):
         for k in multi_out:
             outputs[k] = []
 
-        for i in xrange(self.inputs.n_fibres + 1):
+       for i in xrange(1, self.inputs.n_fibres + 1):
```
Comment (satra) [inline nipype/interfaces/fsl/dti.py]: @mattfeld - could you add the extra space in front of the `for` for indentation.

also could you please update the CHANGES file?

**Comment by mick-d** [comment]
I was just going to submit a PR. This is a small bug but with *huge* consequences. All the pipelines I had on multiple servers were stuck with a ``` "TypeError: 'NoneType' object has no attribute '__getitem__''' ```  I spent a lot of time figuring out where it came from. I would say this is an urgent PR. Any chance to merge it @satra ?

**Comment by satra** [comment]
@mick-d - any chance you can update this PR on aaron's branch with current master so that it can be merged?

**Comment by mick-d** [comment]
Ok i did it but creating a new PR https://github.com/nipy/nipype/pull/1145 on the way, is that all right?

**Pull request merged**